core/scripts/InitDockerConfig.js

```diff
@@ -0,0 +1,31 @@
+const fs = require("fs");
+const util = require("util");
+
+const readFile = util.promisify(fs.readFile);
+const writeFile = util.promisify(fs.writeFile);
+
+const readDockerConfig = async function() {
+  const data = await readFile("./docker-config/config.json");
+
+  return JSON.parse(data);
+};
+
+const processConfig = async function() {
+  const config = await readDockerConfig();
+
+  if ("mnemonic" in config) {
+    let envFile = await readFile("./.env", "utf8");
+
+    // Always append the new parameter to the end of the file.
+    // This overrides an existing parameter, if it exists, without overwriting it.
+    if (envFile.length !== 0 && envFile[envFile.length - 1] != "\n") {
+      envFile += "\n";
+    }
+
+    envFile += `MNEMONIC=${config.mnemonic}\n`;
+
+    await writeFile("../.env", envFile);
```

Comment:
Is there a way we can pass this in memory rather than doubly persisting it on disk? My only concern is that the user may not realize that their mnemonic is exposed in a new file. There may not be a way to do this easily, or it may be too annoying to do as a first cut, just wanted to check.

Reply:
Yeah, I definitely share that concern. @ptare 's commit [here](https://github.com/UMAprotocol/protocol/commit/8502b38e9c3994fbb82ebf3591856ad6c2bdc61f) is essentially passing parameters through memory, but the behavior's defined at build time and not container run time. 

The design isn't ideal so open to suggestions.